src/components/Table/EditableDraggableTable.js

```diff
@@ -181,7 +182,7 @@ class EditableDraggableTable extends React.Component {
       result = value;
     }
 
-    if (!editable && renderConfig && renderConfig.hasAddendum) {
+    if (!editable && additionalData.column.hasAddendum) {
```

Comment:
why not keep it how it was (dependent on renderConfig)?

Reply:
I have column without renderConfig (its not editable) but I want to add addendum to it